jenkinsfiles/ginkgo-gke.Jenkinsfile

```diff
@@ -78,26 +73,32 @@ pipeline {
                 }
             }
         }
-        stage('Log in to dockerhub') {
-            steps{
-                withCredentials([usernamePassword(credentialsId: 'CILIUM_BOT_DUMMY', usernameVariable: 'DOCKER_LOGIN', passwordVariable: 'DOCKER_PASSWORD')]) {
-                    sh 'echo ${DOCKER_PASSWORD} | docker login -u ${DOCKER_LOGIN} --password-stdin'
+        stage('Set programmatic env vars') {
+            steps {
+                script {
+                    if (env.ghprbActualCommit != "") {
+                        env.DOCKER_TAG = env.ghprbActualCommit
+                    } else {
+                        env.DOCKER_TAG = env.GIT_COMMIT
+                    }
+                    if (env.RACE?.trim()) {
+                        env.DOCKER_TAG = env.DOCKER_TAG + "-race"
+                    }
                 }
             }
         }
-        stage('Make Cilium images and prepare gke cluster') {
+        stage('Wait for Cilium images and prepare gke cluster') {
             parallel {
-                stage('Make Cilium images') {
-                    steps {
-                        sh 'cd ${TESTDIR}; ./make-images-push-to-local-registry.sh $(gke/registry-ip.sh) ${TAG} "--no-cache"'
+                stage('Wait for images') {
+                    options {
+                        timeout(time: 10, unit: 'MINUTES')
```

Comment:
The images with `-race` might take longer time before they are available so this timeout needs to be bumped to at least 20 minutes.

Reply:
Good catch.